src/index.js

```diff
@@ -1,2 +1,6 @@
 export { default as Controls, Compass, ZoomControls } from './Controls'
+export { default as BasemapManager, BingMaps, BlankWhite, OpenStreetMap, StamenTerrain, StamenTonerDark, StamenTonerLite } from './Basemaps'
+export { default as Draw, Box, Circle, Freehand, Line, Point, Polygon } from './Draw'
```

Comment:
Should these be namespaced? I feel like exporting `Point` is a bad idea -- what does that even mean? And what if we have another component related to points in the future. If we did `DrawPoint` then at least its more apparent what it is. Same for the basemaps: `BlankWhite` -> `BasemapBlankWhite` or `BlankWhiteBasemap` ?

Reply:
contextualizing is a good idea- just make sure to add a note that this is breaking from pre-release `0.0.0-0`

Reply:
namespacing has been merged to master